examples/asr/conf/citrinet/citrinet_384.yaml

```diff
@@ -11,30 +20,34 @@ model:
     manifest_filepath: ???
     sample_rate: 16000
     batch_size: 32
-    trim_silence: true
+    trim_silence: false
     max_duration: 16.7
     shuffle: true
     is_tarred: false
     tarred_audio_filepaths: null
+    use_start_end_token: false
 
   validation_ds:
     manifest_filepath: ???
     sample_rate: 16000
     batch_size: 32
     shuffle: false
+    use_start_end_token: false
 
   test_ds:
     manifest_filepath: null
     sample_rate: 16000
     batch_size: 32
     shuffle: false
+    use_start_end_token: false
 
   model_defaults:
     repeat: 5
     dropout: 0.0
     separable: true
     se: true
     se_context_size: -1
+    kernel_size_factor: 1.0
```

Comment:
Do you use these model_defaults in the code directly or they are just some place to keep the defaults?

Reply:
model defaults are used in config directly + used by code to know what is globally shared across the configs. Especially useful for RNNT configs. Here, setting the model_defaults.dropout would enable dropout on most of the blocks of Citrinet so its easier for the user.

Reply:
It is fine to have it. Just wanted to make sure that our code is reading these values and set the appropriate places.
If we drop all kernel_size_factor in the rest of the config, does it use this value.

Reply:
Oh no - this value is passed by omegaconf resolution only. Later on this value can be changed but the rest of the config will not be updated. 

Hmm that's a good point, guess I should make it forced propagated via code. But it breaks one niche scenario. Hmm will fix that after some consideration in another pr

Reply:
The option of not propagating is aslo not a bad choice. We would have them as a place holder in the config to get propagated by omegaconf. In this case, no need to worry about conflicts on our side.

Reply:
Yeah, best leave it to omegaconf to handle